browser/ipfs/ipfs_tab_helper.cc

```diff
@@ -39,6 +65,21 @@ void SetupIPFSProtocolHandler(const std::string& protocol) {
   base::MakeRefCounted<shell_integration::DefaultProtocolClientWorker>(protocol)
       ->StartCheckIsDefault(base::BindOnce(isDefaultCallback, protocol));
 }
+
+// Expects dns TXT record in format: name=value
+std::string GetDNSRecordValue(const std::vector<std::string>& text_results,
+                              const std::string& name) {
+  for (const auto& txt : text_results) {
+    if (!base::StartsWith(txt, name))
```

Comment:
seems like it would be safer to split and then check the equality of the name value. I mean it's unlikely someone would do dnslinkblah=whatever, but I also don't really see any benefit to checking StartsWith before splitting